src/test/scala/com/crealytics/spark/v2/excel/LocalFileTestingUtilities.scala

```diff
@@ -55,11 +54,10 @@ trait LocalFileTestingUtilities {
 
     fixture
   }
-
-  def createTemporaryDirectory(name: String): File = {
+  def createTemporaryDirectory(prefix: String): File = {
```

Comment:
Could we use the built-in mechanism for creating temp directories?
https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempDirectory%28java.nio.file.Path,%20java.lang.String,%20java.nio.file.attribute.FileAttribute...%29

Reply:
done